examples/CampaignManagement/ValidateTextAd.php

```diff
@@ -0,0 +1,197 @@
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $adGroupId the ad group ID to validate the ad againts
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $adGroupId
+    ) {
+        // Creates the expanded text ad info.
+        $expandedTextAdInfo = new ExpandedTextAdInfo([
+            'description' => new StringValue(['value' => 'Luxury Cruise to Mars']),
+            'headline_part1' => new StringValue(['value' => 'Visit the Red Planet in style.']),
+            'headline_part2' => new StringValue(['value' => 'Low-gravity fun for everyone!!'])
+        ]);
+
+        // Sets the expanded text ad info on an ad.
+        $ad = new Ad([
+            'expanded_text_ad' => $expandedTextAdInfo,
+            'final_urls' => [new StringValue(['value' => 'http://www.example.com'])]
+        ]);
+
+        // Creates an ad group ad to hold the above ad.
+        $adGroupAd = new AdGroupAd([
+            'ad_group' => new StringValue([
+                'value' => ResourceNames::forAdGroup($customerId, $adGroupId)
+            ]),
+            // Optional: Set the status.
+            'status' => AdGroupAdStatus::PAUSED,
+            'ad' => $ad
+        ]);
+
+        // Creates the ad group ad operation.
+        $adGroupAdOperation = new AdGroupAdOperation();
+        $adGroupAdOperation->setCreate($adGroupAd);
+        $adGroupAdServiceClient = $googleAdsClient->getAdGroupAdServiceClient();
+
+        try {
+            // Try sending a mutate request to add the ad group ad.
+            $response = $adGroupAdServiceClient->mutateAdGroupAds(
+                $customerId,
+                [$adGroupAdOperation],
+                ['validateOnly' => true]
+            );
+
+            // Since validation is ON, result will be null.
+            printf("Expanded text ad validated successfully.%s", PHP_EOL);
+        } catch (GoogleAdsException $googleAdsException) {
+            // This block will be hit if there is a validation error from the server.
+            printf("There were validation error(s) while adding expanded text ad.%s", PHP_EOL);
```

Comment:
Just double check. This fits in 100 chars, right?

Reply:
Yes, it does.